ansible/roles/vm_set/tasks/docker.yml

```diff
@@ -70,13 +70,28 @@
   become: yes
   environment: "{{ proxy_env | default({}) }}"
 
-- name: remove old python packages
-  pip: name=docker-py state=absent executable={{ pip_executable }}
-  become: yes
-  environment: "{{ proxy_env | default({}) }}"
-  ignore_errors: yes
+- name: Update python2 packages
+  block:
+  - name: remove old python packages
+    pip: name=docker-py state=absent executable={{ pip_executable }}
```

Comment:
@lizhijianrd should remove docker-py or docker? This is not same with python3.

Reply:
The old code removes `docker-py` and install `docker`. I guess it want to ensure using `docker`. I update the code to remove `docker-py` for both py2 and py3. @wangxin Can you please help to confirm if this is the expected behavior?

Reply:
Correct. `docker-py` is a very old package used couple of years ago. It has been replaced with `docker` package. This code was to remove the old package during the migration time.